app/src/main/java/com/example/mytestproject/util/CustomBindings.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.mytestproject.util
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import com.example.mytestproject.R
+import com.squareup.picasso.Picasso
+
+
+@BindingAdapter("loadIcon")
+fun bindingImage(weatherIcon: ImageView, iconID: String?) {
```

Comment:
This method is too specific and can be used only for weather icons. Let's make it more common and use it for any image loading. 
So, instead of iconID let's pass imageUrl here. And fix the names to get rid of word Icon.